### PR TITLE
[DNM] the hash slinging slasher (Slot Reduction & AMR Nerf PR)

### PR DESCRIPTION
**Pull request by AllCorruptingWyrm**
## About The Pull Request
A long-time-coming PR has finally been written; This PR slashes NCR slots and BoS slots. Paladins can have their cool armor, maybe? Not that I care personally, its up to the dev team. I know rebels doing a more extensive PR soon but the AMR has got to go.

## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog
:cl:
tweak: weapons and slots
/:cl:

### CODE REVIEW

**Comment by AllCorruptingWyrm** [comment]
yeah so these amr touchups might be a bit longer

**Comment by SingingSpock** [comment]
🧂

**Comment by AllCorruptingWyrm** [comment]
> 🧂

if you have nothing constructive to say then get out of our PR.

**Comment by neri-ventado** [comment]
I seriously don't think that slashing NCR slots will solve whatever issue this is supposed to resolve, if it's trying to. 

If it's trying to fix the constant Legion owning? I think it might be best to wait for that armor PR to go through, since the reason why the teams are so unbalanced is because people don't want to play the Legion.

That's just my thoughts on this.

also atomize ur pr 😜

**Comment by VEN0M5** [comment]
The PR tweaked gauss, hunting revolver, brush guns, scope and bolt action weapons too instead of just AMR, include that in the changelog too. Also you you commented out 50bmg penetrator rounds. Please ensure it is removed form the loot drops

**Comment by SingingSpock** [comment]
This is not actually a solution to any of the problems. It isn't a thoughtful adjustment of the balance it's "buff legion, debuff everyone else without rhyme or reason". Was this discussed with the NCR/Legion lore team?

As others have said, it also majorly needs atomization, slot adjustments and ammo changes should not be in the same PR.

**Comment by AllCorruptingWyrm** [comment]
> The PR tweaked gauss, hunting revolver, brush guns, scope and bolt action weapons too instead of just AMR, include that in the changelog too. Also you commented out 50bmg penetrator rounds. Please ensure it is removed form the loot drops

yes I said this will take forever to look at and is nowhere ready to be done and i'm already in extensive talks with people.
NCR are going to get a nerf, this is not negotiable.

>This is not actually a solution to any of the problems. It isn't a thoughtful adjustment of the balance it's "buff legion, debuff everyone else without rhyme or reason". Was this discussed with the NCR/Legion lore team?
Actually, I've spoken to Tef and rebel about it, they were actually in agreement with things and even zulious has chimed in.

**Comment by VEN0M5** [comment]
I have no clue about balancing, just pointing out potential issues.

**Comment by YosemiteYam** [comment]
The NCR slots being reduced was a needed change for a long time lmao, they literally have like 30 combat roles. It has nothing to do with "oh mer gard you're punishing us for owning legion???". Legion losing is a matter of code as well, with their armor and medicine being broken. Even if they didn't have those problems, they'd be outnumbered by 10 people in a situation of both factions having full slots. 

Bloat and one faction being huge hurts the faction balance and server population cause you make a single military faction consist of half the server, even though it can't sustain a roleplay center for others due to its position as a military faction with its own goals and hostility, thereby just creating an enclave of a large amount of ncr rping amongst themselves.

**Comment by neri-ventado** [comment]
Again, I think it's a result of the whole healing-and-armor being fucked by code. There is time where the Legion |does| win, though - and that's when they actually group together, and snake up.

The Legion right now never do that!! They send explorer after explorer, by themselves, and it's a trickle down effect.

image

Also, the NCR have eight combat roles, while the Legion have ten. Again, it's an issue of people not being Legion roles themselves.

**Comment by YosemiteYam** [comment]
Firstly its more an issue of player slots, not number of unique roles. NCR has more more slots available, period.

Secondly I think you need to count abit better; There's literally 10 NCR combat ROLES in the picture you shared (Captain, LT, Heavy Trooper, Sergeant, Corporal, Specialist, Trooper, Conscript, Vet Ranger, Ranger) and 9 legion combat roles (Venator doesn't exist its just on the list for reason). This also doesn't take into the fact that unlike the non combatants in the legion, rear echelon troops frequently participate in combat/leave the base instead of acting as off duty/base garrison.

Legion losing is because of code (armor, meds being broken) but its also to a lesser extent because of unfair numbers balancing, and regardless of whether they were losing or not is irrevelent, a faction doesn't deserve to have over 30 player slots "just because" when the other factions don't approach that number and it hurts the stability of a healthy playerbase.

**Comment by AllCorruptingWyrm** [comment]
ncr slots are being adjusted 

please dont argue in my dms, post here instead.
I'm going to bed.

**Comment by Teragenic** [comment]
My comments: Reducing Rear echelon slots AND also specialists.

While reducing specialists is fair, they are not always the 'meta' combat rocket RPG loadout. They might be engineers or medics, both mostly non-combat roles.

Doing this just reduces the 'support' roles and not the actual combat roles. Atleast for NCR. 

The removal of Sergeants to 1 and reducing corporal to /2/ seems a bit WHILE increasing Primes to /5/ and Vet Legionaries to /4/ is a bit much. 

I'd instead just say increase Recruit/Decanus slots to /2/ to also add more 'leading' power to the legion, instead of nerfing Sergeants and Corporals to essentially forcing them to be 'combat' roles. Despite them also being support AND this.

**Comment by BadAtThisGame302** [comment]
@AllCorruptingWyrm So you remove A Knight/Scribe/Initiate slot... but not a Paladin one, along with reducing SGTs from 3 to 1 (LOL, ideally a 1 SGT would lead 2 Troopers, a Spec and a Corporal, now the 1 SGT leads 2 Specs, 3 Troopers and 2 Corporals...) 
Halving specialists (I guess this is fine, Engie spec is OP)
HALVING NCR Trooper (LOL, again, what is this, the NCR is meant to have a grunt force, it is not a topdown system like the Legion) 
Reducing the Rear Ech slot which is used as an off-duty replacement (I have seen a lot of NCR Rangers, especially, go Wastelander and come to the NCR Base and LOOC there's no Rear Ech slots)
Reducing Ranger slots (I can't speak much about this, I guess it's alright if it weren't bunged in with the rest of the changes)

If I also read this right, this reduces NCR slots to 23 (16 of which are combat roles) ((MO and Rear Ech removed)) and INCREASES Legion slots to 30 and their combat slots to 23 (The NCRs total SLOTS, LOL) ((Auxilia and Camp Follower removed))

Along with bundling gun changes to major faction balance changes... yeah.. close this PR.

**Comment by BadAtThisGame302** [comment]
> The NCR slots being reduced was a needed change for a long time lmao, they literally have like 30 combat roles. It has nothing to do with "oh mer gard you're punishing us for owning legion???". Legion losing is a matter of code as well, with their armor and medicine being broken. Even if they didn't have those problems, they'd be outnumbered by 10 people in a situation of both factions having full slots.
> 
> Bloat and one faction being huge hurts the faction balance and server population cause you make a single military faction consist of half the server, even though it can't sustain a roleplay center for others due to its position as a military faction with its own goals and hostility, thereby just creating an enclave of a large amount of ncr rping amongst themselves.

Currently NCR have 33 slots while Legion have 27, with the proposed slot changes NCR will lose ten slots and Legion will get 3... the disparity is bigger than the current one, very balanced.

File: code/__DEFINES/combat.dm [hunk 1]
```diff
@@ -584,7 +584,7 @@ GLOBAL_LIST_INIT(main_body_parts, list(
 #define RUBBERY_RECOIL_RIFLE_4570 (BULLET_RECOIL_RIFLE_4570 * RUBBERY_RECOIL_MULT)
 #define RUBBERY_WOUND_RIFLE_4570 (BULLET_WOUND_RIFLE_4570 * 4)
 
-#define BULLET_DAMAGE_RIFLE_50MG 75
+#define BULLET_DAMAGE_RIFLE_50MG 48
```
Comment (BadAtThisGame302) [inline code/__DEFINES/combat.dm:587]: I am glad that the AMR does LESS damage than the 45/70 now. Sounds like the weapon that can pierce Power Armor.

**Comment by Jul-Jul** [comment]
I could possibly agree with the nerfing of AMR since everyone and their mothers can get hold of one from Vault printing them. Which soon will be removed... so it will become a rear weapon again. Thus somewhat invalidating their nerf.

Secondly, these slots changes are way too drastic right before multiple major changes such as the armor rebalance and moving to Tipton. Sure give Legion to same slots as NCR, but don't just remove NCR and BoS, as it just seems you're punishing NCR and BoS because Legion ain;t played. Alongside the fact that removing of slots for a server that reaches 80-90 in prime time is, simply idiotic. You're making it unlikely for people to play the factions that they want, which will just push them from the server.

**Comment by AllCorruptingWyrm** [comment]
It's happening but i'm going to close this for now.

**Pull request closed without merging**